bin/NativeTests/JsRTApiTest.cpp

```diff
@@ -1472,6 +1474,48 @@ namespace JsRTApiTest
         JsRTApiTest::RunWithAttributes(JsRTApiTest::PromisesTest);
     }
 
+    void UnsetPromiseContinuation(JsRuntimeAttributes attributes, JsRuntimeHandle runtime)
+    {
+        JsValueRef result = JS_INVALID_REFERENCE, callbackState = JS_INVALID_REFERENCE, exception = JS_INVALID_REFERENCE;
+        JsValueType cbStateType = JsUndefined;
+
+        // script with no promise continuation callback should error
+        REQUIRE(JsRunScript(_u("new Promise((res, rej) => res()).then(() => 1)"), JS_SOURCE_CONTEXT_NONE, _u(""), &result) == JsErrorScriptException);
+        CHECK(result == JS_INVALID_REFERENCE);
+        REQUIRE(JsGetAndClearException(&exception) == JsNoError);
```

Comment:
Perhaps also verify that the exception that was reported matches the one that we expect?